flake.nix

```diff
@@ -22,37 +32,79 @@
             openssl
             pkg-config
             perl
+            fenix-channel.rustc
           ];
 
           nativeBuildInputs = with pkgs; [
             pkg-config
           ];
         };
 
-        cargoArtifacts = craneLib.buildDepsOnly (commonArgs // {
-          pname = "minimint";
+        minimintDeps = craneLib.buildDepsOnly (commonArgs // {
```

Comment:
I don't really like the default `crane` naming with `cargoArtificats`. Sure it makes possible to do `inherit cargoArtifacts;` which is shorter, but at the cost of clarity.